gatsby_fp/src/components/fieldComponents/genInfoTable.js

```diff
@@ -0,0 +1,122 @@
+import React from "react"
+
+export default function GenInfoTable({
+  school,
+  title,
+  imgsrc,
+  building,
+  room,
+  phone,
+  fax,
+  username,
+  website,
+}) {
+  return (
+    <>
+      <div className="table_wrapper">
+        <div className="table_wrapper_inner">
+          <table
+            cellPadding="5"
```

Comment:
It's how things were before, so I won't call this blocking, but we should create another issue to move all of these inline styling attributes into CSS. Markup should control semantic structure, not the presentation itself.